platform/cavium/cavm_platform_modules/Makefile

```diff
@@ -0,0 +1,27 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = cavm_platform_modules.deb
+DEB_BUILD_DIR = cavm-platform-modules-deb
+KVERSION = linux-3.16.36
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# get sources
+	rm -rf SONiC
+	git clone https://github.com/edge-core/SONiC.git
+
+	# build
+	pushd SONiC/AS7512-32X/module/
+
+	KERNEL_SRC=/sonic/src/sonic-linux-kernel/$(KVERSION)/debian/build/build_amd64_none_amd64 make
```

Comment:
@NStetskovych Kernel release version is unchanged 3.16.0-4-amd64 and should be used for cross-compilation without caring about distro versions.

@lguohan I already wanted to install them, but I didn't find any difference between headers provided by those two distro versions and what we have now. Anyway, it will be better to install them to ensure we won't miss anything in the future. There's going to be a separate PR with headers integrated and variables for distro version and subversion to avoid duplication as it is done for other targets.